modules/fedimint-ecash-migration-common/src/naive_threshold.rs

```diff
@@ -1,148 +0,0 @@
-//! Naive threshold signature scheme implementation using concatenated Schnorr
```

Comment:
Ohhhh... That's why merkle tree is used. Because we got rid of threshold sigs ... . So this is now our auth...